### PR TITLE
Give math input keyboard logical focus order

### PR DESCRIPTION
**Pull request by nishasy**
Summary:
- Gave all keys tabIndex 0 so they can be in focus
- Reordered keys so they're listed in tab order
- Used CSS to make the keys appear in the right place visually

Issue: https://khanacademy.atlassian.net/browse/LP-7269

### CODE REVIEW

File: src/components/fraction-keypad.js [hunk 1]
```diff
@@ -78,95 +88,116 @@ class FractionKeypad extends React.Component {
             dismissOrJumpOutKey = KeyConfigs.DISMISS;
         }
 
-        return <Keypad>
```
Comment (nishasy) [inline src/components/fraction-keypad.js]: Note: Since it does look like the expression keypad and the fraction keypad share code, I tried to create a new "keypad-segments" file that contains the separate components to be reused. However, it turns out that there are a bunch of little differences that make this less practical than it sounds. (e.g. styling on the 7 button, % button in place of cdot, minus button in place of variable key)

I decided to leave it and just wait for the new redesign to start development first.

File: src/components/expression-keypad.js [hunk 2]
```diff
@@ -4,10 +4,10 @@
 
 const React = require('react');
 const PropTypes = require('prop-types');
-const {connect} = require('react-redux');
-const {StyleSheet} = require('aphrodite');
+const { connect } = require('react-redux');
```
Comment (dierat) [inline src/components/expression-keypad.js]: Could you do me a huge favor and pull the prettier changes out into a different PR?

File: src/components/keypad.js [hunk 3]
```diff
@@ -121,7 +122,7 @@ class Keypad extends React.Component {
         };
 
         return <View style={style}>
-            {children}
+            {active && children}
```
Comment (dierat) [inline src/components/keypad.js]: Can you explain this change for me please?

File: src/components/keypad-container.js [hunk 4]
```diff
@@ -181,6 +181,8 @@ class KeypadContainer extends React.Component {
             extraClassName="keypad-container"
         >
             <View
+                // List in logical tab order for a11y and
+                // correct visual order with styling
```
Comment (dierat) [inline src/components/keypad-container.js]: Thank you so much for this comment! <3

File: src/components/keypad-button.js [hunk 5]
```diff
@@ -218,6 +218,7 @@ class KeypadButton extends React.PureComponent {
             const manyButtonA11yMarkup = {
                 role: 'button',
                 ariaLabel: childKeys[0].ariaLabel,
+                tabIndex: 0
```
Comment (dierat) [inline src/components/keypad-button.js]: w00t

File: src/components/view-pager.js [hunk 6]
```diff
@@ -83,11 +83,12 @@ class ViewPager extends React.Component {
         };
 
         return <View style={pagerStyle} dynamicStyle={dynamicPagerStyle}>
+            {/* list the number keypad page first for tab order */}
```
Comment (dierat) [inline src/components/view-pager.js]: Love the comments!

File: src/components/two-page-keypad.js [hunk 7]
```diff
@@ -44,13 +44,13 @@ class TwoPageKeypad extends React.Component {
             </Keypad>;
         } else {
             return <Keypad style={styles.keypad}>
-                <View style={row}>
-                    <View style={fullWidth}>
-                        {leftPage}
-                    </View>
+                <View style={reverseRow}>
                     <View style={[styles.borderLeft, fullWidth]}>
```
[the remark this answers is not in the document]
Comment (nishasy) [reply src/components/two-page-keypad.js]: The right page contains the numbers, and the left page contains additional math operators. While it shows up with the operators on the left and the numbers on the right, we want the numbers to receive focus first when tabbing through the keyboard.

File: src/components/keypad.js (hunk 3 above)
Comment (nishasy) [reply src/components/keypad.js]: I added this so that the individual buttons only show up in the DOM if the keyboard has been activated. It was doing a thing where even when the keyboard wasn't on the page, all the buttons were showing up in the form controls rotor in the screen reader.

I'll add a comment in the code for clarity!
